### DISCUSSION TITLE
MX Master 2S Gesture Button

### DISCUSSION THREAD

**Discussion by drew5494** [noah-nuebling/mac-mouse-fix#1079] [Questions]
This is a really great app. 
I am using it with my Logitech mouse. It has a gesture button but can't seem to get it working. Any ideas? [
![](http://www.logitech.com/assets/53449/2/mx-master-and-logitech-options.jpg)
](url)

**Comment by enryf89** [comment]
same issue here. I don't like Logitech Options + and would love to use Mac Mouse Fix, but if it doesn't support the "gesture Button" it doesn't really make sense.

**Reply by ccarden-git** [reply]
the opinion. i have a logitech M590, that similar as the MX can be used in different devices. Therefore it would be genial if one can configurate it with all the available buttons
